what

```diff
@@ -50,7 +57,7 @@ for pid in os.listdir("/proc"):
         ttys[tty_nr][-1].append(cmdline.replace("\0", " "))
 
 # Sort TTYs by input time
-ttys = sorted(ttys.values(), key=lambda tty: tty[1].st_atime)
+ttys = sorted(list(ttys.values()), key=lambda tty: tty[1].st_atime)
```

Comment:
In this case `list` is unnecessary: in python2 `sorted(ttys.values())` creates a list and then a new sorted version of that list, whereas in python3 `sorted(ttys.values())` creates an iterator that `sorted` turns into a sorted list.